docs/blockchain_schema/version1.ts

```diff
@@ -0,0 +1,61 @@
+// Version 1 schema
+
+type Multihash = string;
+type HashPointer = string;
+type UID = string;
+type Datetime = string;
+
+// Block schema
+interface Block {
+  sequence: number; // 0-indexed
+  // The datetime at which the earliest version of this block was created.
+  origin_created: Datetime; // new
+  version: 1;
+  // Points to the same block in the previous version of the blockchain.
+  prev_version_block?: HashPointer; // new
```

Comment:
This is optional, but `prev_hash` below is required but can be `null`. Unless there's a good reason for the inconsistency, I'd rather we are consistent about this sort of thing.

`null` seems to make it explicit that the field will be there but it may have a `null` value. Optional makes it seem like a valid block might actually be missing this member altogether.

Reply:
My reasoning for that was that prev_hash should always be there, and it is only null in the case of the origin block.

prev_version_block only needs to be present if this block was present in a previous version of the blockchain, which will not be true for most new blocks, especially as the schema stabilizes.

Does that seem reasonable?